docs/topics/wildfly-swarm/docs/reference/fractions/undertow.adoc

```diff
@@ -287,7 +344,10 @@ swarm.undertow.handler-configuration.reverse-proxies._KEY_.connection-idle-timeo
 The amount of time a connection can be idle before it will be closed. Connections will not time out once the pool size is down to the configured minimum (as configured by cached-connections-per-thread)
 
 swarm.undertow.handler-configuration.reverse-proxies._KEY_.connections-per-thread:: 
-The number of connections that will be maintained to backend servers, per IO thread. Defaults to 10.
+The number of connections that will be maintained to backend servers, per IO thread.
+
+swarm.undertow.handler-configuration.reverse-proxies._KEY_.hosts._KEY_.enable-http2:: 
+If this is true then the proxy will attempt to use HTTP/2 to connect to the backend. If it is not supported it will fall back to HTTP/1.1/
```

Comment:
Should 

> HTTP/1.1/

be 

> HTTP 1.1

or something else?

Reply:
Note to self: not easily fixable in our sources.